src/crypto/algorithms/megolm.ts

```diff
@@ -589,8 +589,7 @@ class MegolmEncryption extends EncryptionAlgorithm {
      *
```

Comment:
This file has new code: updated to not use the return value of `encryptAndSendToDevices ` as it already had all the info it needed.